docker/hoodie/hadoop/prestobase/bin/mustache.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# `mustache.sh`, Mustache in POSIX shell.
```

Comment:
is this based off some script int he open like the file above?

Reply:
Not aware of the open source script. But this one seems to be referring to the script under the lib directory - docker/hoodie/hadoop/prestobase/lib/mustache.sh. 

From what I understood, this script is used for templating common props and passing in params  when needed, to create specific instances. For example, in this context, it is leveraged to create either a coordinator or worker instance from the same Presto image by passing in the relevant properties file as params.